pkg/trace/api/api.go

```diff
@@ -339,6 +339,10 @@ const (
 	// headerComputedStats specifies whether the client has computed stats so that the agent
 	// doesn't have to.
 	headerComputedStats = "Datadog-Client-Computed-Stats"
+
+	// headderDroppedP0s contains the number of P0 trace chunks dropped by the client.
+	// This value is used to adjust priority rates computed by the agent.
+	headerDroppedP0s = "Datadog-Client-Dropped-P0s"
```

Comment:
Can you change this to "Datadog-Client-Dropped-P0-Traces" so we can also add "Datadog-Client-Dropped-P0-Spans" for the future?